deployment/nginx.conf.template

```diff
@@ -15,6 +15,7 @@ http {
       proxy_http_version 1.1;
       proxy_set_header Upgrade $http_upgrade;
       proxy_set_header Connection "upgrade";
+      client_max_body_size 100m;
```

Comment:
Can this be added to the config and made a variable?

Reply:
Yes it could be made a variable using the same method we use for e.g. CLIENT_PORT.
Unless this is parity I'd assume it needs to be kicked down the road?

Reply:
Let's get this in for now - this is part of the config tech debt